src/app/user-registration-form/user-registration-form.component.html

```diff
@@ -0,0 +1,152 @@
+<mat-card class="authentication-card">
+    <mat-card-title>Summer Good Deeds</mat-card-title>
+    <mat-card-subtitle>Registration</mat-card-subtitle>
+    <div class="registration-fields">
+        <form (ngSubmit)="registerUser()" [formGroup]="registerForm">
+            <div class="registration-form-field">
+                <mat-form-field appearance="outline" class="register-form-field">
+                    <mat-label>First name</mat-label>
+                    <input formControlName="name" matInput placeholder="" required maxlength="20" />
```

Comment:
same thing with the max length constants

Reply:
+